src/screens/WhotoFollow/elements/UserInfo.js

```diff
@@ -1,8 +1,10 @@
 import * as React from 'react';
 import {Image, StyleSheet, Text, View} from 'react-native';
 import PropTypes from 'prop-types';
-import {normalize} from '../../../utils/fonts';
+import {normalize, normalizeFontSize} from '../../../utils/fonts';
 import {CircleGradient} from '../../../components/Karma/CircleGradient';
+import {COLORS} from '../../../utils/theme';
+import dimen from '../../../utils/dimen';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for maintainability and consistency. The use of `normalize` and `normalizeFontSize` from the `utils/fonts` file, as well as `dimen` from the `utils/dimen` file, ensures that the dimensions and font sizes used in the application are consistent across different screen sizes and resolutions. Also, importing `COLORS` from the `utils/theme` file helps to maintain a consistent color scheme throughout the app.
```diff
- import {normalize} from '../../../utils/fonts';
+ import {normalize, normalizeFontSize} from '../../../utils/fonts';
+ import {COLORS} from '../../../utils/theme';
+ import dimen from '../../../utils/dimen';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->